GitCommands/Git/GitRevisionSummaryBuilder.cs

```diff
@@ -21,6 +21,11 @@ public class GitRevisionSummaryBuilder : IGitRevisionSummaryBuilder
 
         public string BuildSummary(string body)
         {
+            if (body == null)
```

Comment:
What about empty string or whitespace?

Reply:
I imagine because the empty or whitespace is well handled by the code and I just wanted to avoid the NRE.

Reply:
If `body` is empty or whitespace we will be making allocations and burning cycles. Hence my question - is there a need to do that?

Reply:
This is used only for tooltips (currently). Hence I do not expect performance issues.
In general, I prefer to display exactly what's in git.

Reply:
In fact, I think that's a false debate and that there will never be an empty or whitespace git commit message.

I only wished to prevent a NRE because, for a reason, a tooltip is generated when the git message has the null value (which is counter-intuitive).  

So, I imagine that in fact, we could add `string.IsNullOrWhiteSpace()`, it will change nothing...

Reply:
@RussKie I have pushed the string.IsNullOrWhiteSpace().

Feel free to squash on merge.